index.js

```diff
@@ -0,0 +1,119 @@
+var days = [
+    'SUN',
+    'MON',
+    'TUE',
+    'WED',
+    'THU',
+    'FRI',
+    'SAT'
+];
+
+var months = [
+    'JAN',
+    'FEB',
+    'MAR',
+    'APR',
+    'MAY',
+    'JUN',
+    'JUL',
+    'AUG',
+    'SEP',
+    'OCT',
+    'NOV',
+    'DEC'
+];
+
+var monthdays = [31,28,31,30,31,30,31,31,30,31,30,31];
+
+var t = new Date();
+var tdate = t.getDate();
+var tday = t.getDay();
+var tmonth = t.getMonth();
+var tyear = t.getFullYear();  // 앞에 t 붙어있는거 -> 현재 날짜 가져옴.
+
+var d = tdate;
+var m = tmonth;
+var y = tyear;
+var nday = new Date(y,m,d).getDay();
+var arr = [];  // 날짜 저장
+
+function showDays(){
+
+    document.querySelector('.year').innerHTML = y;
+    document.querySelector('.dal').innerHTML = months[m];
+    document.querySelector('.date').innerHTML = d;
+    document.querySelector('.days').innerHTML = days[nday];
+
+    var lastday = monthdays[m];  // 이 달의 마지막 날
+    var firstday = new Date(y,m).getDay(); // 이 달의 첫번째 요일, 요일수만큼 0이 앞에 생긴다.
+    var count = "";
+
+    for (j=0; j<firstday; j++){
+        count += '<div class="zeros">&nbsp</div>';
+        document.querySelector('.numbers').innerHTML = count;
+        arr.push(0);
+    }
+
+    for (i=1; i<= lastday; i++){
+         if(i==tdate && m==tmonth && y==tyear){
+            count += '<div style="color:red" id =' + i + '>' + i + '</div>';
+        }
+        else{
+            count += '<div id =' + i + '>' + i + '</div>';
+        }
+        document.querySelector('.numbers').innerHTML = count;
+        arr.push(i);
+    }
+}
+
+var u = document.querySelector('.numbers'); // 달력 클릭시 표시 날짜 바꾸기 // 구현안됨
+u.addEventListener('click', function(event){
```

Comment:
해당 함수 안에서 event는 사용하지 않기때문에 생략해도 됩니다.
```suggestion
u.addEventListener('click', function(){
```

남겨주신 메세지에서 왜 생략 가능한지 궁금하신것 같아 추가로 글 남깁니다. 

함수는 선언과 실행 두가지 동작으로 나뉩니다. 
이곳에서는 함수를 선언하고 있습니다.
클릭이라는 이벤트가 발생했을때 해당 함수는 실행 됩니다. 
실행되면서 첫번째 인자로 브라우저상에 존재하는 event객체를 넘깁니다. `listener(event);`내부적으로 실행됩니다. 볼 수 없습니다. listener는 수정님이 선언한 함수입니다. 
선언된 함수(listener)에서 첫번째 인자로 event를 받게 되고 해당 인자를 사용할지 말지는 함수를 선언한 사람이 결정하면 됩니다. 
따라서 event를 적던 적지 않던 같은 동작을 하게 됩니다. 

이러한 함수를 callback함수라고 합니다. 
굉장히 중요하지만 처음 접하면 어려울 수 있습니다. 
이 부분은 차근차근 배우게 되실거에요 걱정하지 않으셔도 됩니다 😊

Reply:
상세하게 리뷰 & 설명해주셔서 감사합니다! 😃